data/quickstarts/create-ml-pipeline-quickstart.yaml

```diff
@@ -0,0 +1,92 @@
+kind: ConsoleQuickStart
+metadata:
+  name: create-ml-pipeline
+  annotations:
+    opendatahub.io/categories: 'Getting started,ML Pipelines'
```

Comment:
As above, 
Please refer to the list of categories in the README file added in PR#224 to select appropriate categories here. https://github.com/red-hat-data-services/odh-dashboard/pull/224/files#diff-5b17ec93fc259f65542b5ec813578981a6104c4f8238a6006c5cd2a9e92f6786

'Getting started,Model serving' is probably correct here but let me know if we should add another category for this type of app / work.